### ISSUE TITLE
Incorrect file format: expected parameter password not found

### ISSUE THREAD

**Issue by Cempakers87** [bbernhard/signal-cli-rest-api#92]
Hello,
I have tried to register my dedicated number for this api with your example in https://github.com/bbernhard/signal-cli-rest-api/issues/19#issuecomment-772815209 but error occured everytime I try to register (see picture).
![error signal rest api](https://user-images.githubusercontent.com/20413727/107758487-8670fa00-6d59-11eb-8f79-2d230de79f66.png)
Did I miss something? Please help. Thank you. :)

**Comment by bbernhard** [comment]
That error looks really strange - haven't seen that one before. I guess you are using a docker-compose file similar to the one in the `README`? Are you sure that you have mounted a directory inside the docker container that is writable from within the container?

**Comment by Cempakers87** [comment]
Hi @bbernhard, thank you for your quick response. Yes I'm sure I have mounted that directory, FYI, I have tried to register different number about 1 week ago with success.

After I look deep inside container and `cd /home/.local/share/signal-cli/data `, I found my number there as a JSON file. If you `cat` that file, you will find password prop with empty value. I try to delete that file and re-register with command I mentioned above, it works. Now, I can use the API.

Thank you.
